ppr-ui/src/resources/tableHeaders.ts

```diff
@@ -441,56 +441,56 @@ export const registrationTableHeaders: Array<BaseHeaderIF> = [
 export const mhRegistrationTableHeaders: Array<BaseHeaderIF> = [
   {
     class: 'min-column-md',
-    sortable: false,
+    sortable: true,
```

Comment:
Same question for all here, using the Vuetify table sorting out of the box works?
No mapped value issues?

Reply:
I believe it uses the Vuetify sortable to make the header clickable and display the sort direction arrow, but the actual sort is done by orderBy in script. Please correct me if I'm wrong, but seems to be no mapping issues!

Reply:
Sorry yes, i wasn't very clear, you are correct! This enables the button -> in turn the event -> in turn the function. 
Was definitely referring to the sort function itself, but if it works as is, thats great!